Tools/test/TrackingTreer/hists.cc

```diff
@@ -2,12 +2,19 @@
 #include "JMTucker/MFVNeutralino/interface/Ntuple.h"
 #include "JMTucker/Tools/interface/NtupleReader.h"
 
+
 int main(int argc, char** argv) {
   jmt::NtupleReader<jmt::TrackingAndJetsNtuple> nr;
-  nr.init_options("tt/t", "TrackingTreerHistsV23mv3", "nr_trackingtreerv23mv3", "ttbar=False, all_signal=False");
+  // nr.init_options("tt/t", "TrackingTreerHistsV23mv3", "nr_trackingtreerv23mv3", "ttbar=False, all_signal=False");
+  //need to include scale type 
+  nr.init_options("tt/t", "TrackingTreerULV2_Lepm_cut0_etalt1p5_2018_wsellep", "trackingtreerulv2_lepm", "ttbar=False, leptonic=False, all_signal=False, qcd_lep=False, met=False, diboson=False, Lepton_data=False ");
+  //nr.init_options("tt/t", "TrackingTreerULV1_Lepm_etagt1p5_eraF_2017", "trackingtreerulv1_lepm_wsellep", "ttbar=False, leptonic=True, all_signal=False, qcd_lep=True, met=True, diboson=True, Lepton_data=False ");
```

Comment:
@abbywarden I think this was yours, right? I think we could delete a lot of the commented out stuff (maybe also in cov_ratio_v_pt.py, but its more clear cut here). Also some other large commented areas here in hists.cc (for other TH1/TH2s and for an "old" method), that could be worth cleaning up.